src/Capgemini.PowerApps.PackageDeployerTemplate/Services/ProcessActivatorService.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Capgemini.PowerApps.PackageDeployerTemplate.Adapters;
+using Capgemini.PowerApps.PackageDeployerTemplate.Extensions;
+using Microsoft.Extensions.Logging;
+using Microsoft.Xrm.Sdk;
+using Microsoft.Xrm.Sdk.Query;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Capgemini.PowerApps.PackageDeployerTemplate.Services
+{
+    public class ProcessActivatorService
```

Comment:
Might be a good idea to widen the scope of this one for future functionality e.g. `ProcessDeploymentService`